### DISCUSSION TITLE
bing map language

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#1406] [Q&A] [answered]
when i check Map Labels,and the road  show ,but i want the road name show by chinese language。
![QQ截图20210517135122](https://user-images.githubusercontent.com/18022720/118438577-2ce0e880-b717-11eb-93f1-d4b360680fc0.jpg)

**Comment by pmconne** [comment, marked as answer]
Bing Maps [does not supply](https://docs.microsoft.com/en-us/bingmaps/rest-services/common-parameters-and-types/supported-culture-codes) imagery with Chinese labels.
